Mage.Server.Plugins/Mage.Tournament.BoosterDraft/src/mage/tournament/cubes/LegendaryCubeApril2016.java

```diff
@@ -37,7 +37,7 @@
 
     public LegendaryCubeApril2016() {
         super("Legendary Cube April 2016 (566 cards)");
-        http://magic.wizards.com/en/MTGO/articles/archive/legendary-cube-cardlist
```

Comment:
This URL just happened to not be causing issues... Java interprets `http:` as a tag, and the following `//...` is interpreted as a comment